fbw-a380x/src/wasm/systems/a380_systems/src/air_conditioning/mod.rs

```diff
@@ -266,6 +336,22 @@ impl A380AirConditioningSystem {
                     ElectricalBusType::AlternatingCurrent(2),
                 ],
             ),
+            fdac: [
+                FullDigitalAGUController::new(
+                    1,
+                    vec![
+                        ElectricalBusType::DirectCurrentEssential, // 403XP
+                        ElectricalBusType::AlternatingCurrent(1),  // 117XP
+                    ],
+                ),
+                FullDigitalAGUController::new(
+                    2,
+                    vec![
+                        ElectricalBusType::DirectCurrentEssential, // 403XP
+                        ElectricalBusType::AlternatingCurrent(2),  // 204XP
```

Comment:
```suggestion
                        ElectricalBusType::AlternatingCurrent(4),  // 204XP
```
204XP is powered by AC BUS 4.
Now that the electrical system is merged you can use AC BUS 4 ;)